### PR TITLE
Added v9 guide to add 404 error page

### PR DESCRIPTION
**Pull request by nikcio**
I've added a v9 guide on how to add a 404 page. I copy-pasted the current document and filled out the 404 section as I still don't know how to do the rest documented in the current document. I figured some v9 documentation was better than nothing.

### CODE REVIEW

**Comment by jonat123** [comment]
@nikcio Looks good! Thanks for the PR 😄

We will make sure to review it asap!

File: Tutorials/Custom-Error-Pages/index-v9.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+In this method we will use a 404 page created via the backoffice.
+
+### Create a 404 page in the backoffice
+
+First, create a new document type (though you could also use a more generic document type if you already have one) called Page404.
+Make sure the permissions are set to create it under Content.
+Properties on this document type are optional - in most cases, the 404 not found page would be static.
+Make sure to assign (and fill out) the template for your error page, and then create it in Content.
+
+### Set a custom 404 page in umbracoSettings.config
```
Comment (jonat123) [inline Tutorials/Custom-Error-Pages/index-v9.md:33]:
```suggestion
### Set a custom 404 page in appsettings.json
```

Comment (nikcio) [reply Tutorials/Custom-Error-Pages/index-v9.md]: Great I missed that. I've added it now

**Comment by jonat123** [comment]
@nikcio Just went through the article and tested it out.

It works like a charm, let's get this merged in 😄

I can also see that this is your first contribution to the Umbraco Documentation, therefore you are entitled to a shiny C-tribe badge on your Our.Umbraco Profile! If you can let me know your user-name, then i will get it assigned to you 😄

**Comment by nikcio** [comment]
@jonat123 I already have the C-Trib badge from contributing to the UmbracoCms project. But thank you.

See here: https://our.umbraco.com/members/nikcio/

**Pull request merged**